pandas/io/json/_json.py

```diff
@@ -145,74 +145,82 @@ def _format_axes(self):
         raise AbstractMethodError(self)
 
     def write(self):
-        return self._write(
+        iso_dates = self.date_format == "iso"
+        return dumps(
             self.obj,
-            self.orient,
-            self.double_precision,
-            self.ensure_ascii,
-            self.date_unit,
-            self.date_format == "iso",
-            self.default_handler,
-            self.indent,
+            orient=self.orient,
+            double_precision=self.double_precision,
+            ensure_ascii=self.ensure_ascii,
+            date_unit=self.date_unit,
+            iso_dates=iso_dates,
+            default_handler=self.default_handler,
+            indent=self.indent,
         )
 
-    def _write(
+
+class SeriesWriter(Writer):
+    _default_orient = "index"
+
+    def __init__(
         self,
         obj,
         orient: Optional[str],
+        date_format: str,
         double_precision: int,
         ensure_ascii: bool,
         date_unit: str,
-        iso_dates: bool,
-        default_handler: Optional[Callable[[Any], JSONSerializable]],
-        indent: int,
+        index: bool,
+        default_handler: Optional[Callable[[Any], JSONSerializable]] = None,
+        indent: int = 0,
     ):
-        return dumps(
+        super().__init__(
             obj,
-            orient=orient,
-            double_precision=double_precision,
-            ensure_ascii=ensure_ascii,
-            date_unit=date_unit,
-            iso_dates=iso_dates,
+            orient,
+            date_format,
+            double_precision,
+            ensure_ascii,
+            date_unit,
+            index,
             default_handler=default_handler,
             indent=indent,
         )
-
-
-class SeriesWriter(Writer):
-    _default_orient = "index"
+        if not index and orient == "split":
+            self.obj = {"name": obj.name, "data": obj.values}
```

Comment:
Hmm why do we need to do this now?

Reply:
I do not know. I simply used the original implementation and moved whatever assignments were in ``_write`` into ``__init__``.

Reply:
Oh OK - I see what was done here. Let's not assign to `self.obj` as that will mess with the idempotency

Reply:
We assign to ``self.obj`` at the moment of the writer initialization, not when calling a method. So, I would argue that it is not harmful.

Reply:
I still think it is confusing to change what `self.obj` is for certain orients - would prefer not to do this